docs/components/typography/test/index.html

```diff
@@ -0,0 +1,299 @@
+    <!-- Syntax Highlighting -->
+    <figure data-dom-diff="typography-syntax-bash">
+        <figcaption>bash Syntax Highlighting Example</figcaption>
+        {% code 'bash' %}
+-        #!/bin/bash
+-
+-        ###### CONFIG
+-        ACCEPTED_HOSTS="/root/.hag_accepted.conf"
+-        BE_VERBOSE=false
+-
+-        if [ "$UID" -ne 0 ]
+-        then
+-          echo "Superuser rights required"
+-          exit 2
+-        fi
+-
+-        genApacheConf(){
+-          echo -e "# Host ${HOME_DIR}$1/$2 :"
+-        }
+-      {% endcode %}
+    </figure>
+</section>
+<section>
+    <!-- Syntax Highlighting -->
+    <figure data-dom-diff="typography-syntax-css">
+        <figcaption>CSS Syntax Highlighting Example</figcaption>
+        {% code 'css' %}
+-        /* CSS Comment */
+-        @import url('https://fonts.googleapis.com/css?family=Roboto+Mono:100,100i,300,300i,400,400i,700,700i');
+-        :root {
+-          --border-color: #bada55;
+-        }
+-        html {
+-          box-sizing: border-box;
+-        }
+-        *,
+-        *::before,
+-        *::after {
+-          box-sizing: inherit;
+-        }
+-        #navigation {
+-          border: 1px solid var(--border-color);
+-          padding: 1em;
+-          margin: 0;
+-          background-image: linear-gradient(90deg, rgba(0, 0, 0, 0,5), white);
+-        }
+-        input[type="checkbox"]:checked {
+-          /* ... */
+-        }
+-        .text-center {
+-          text-align: center !important;
+-        }
+-      {% endcode %}
+    </figure>
+</section>
+<section>
+    <!-- Syntax Highlighting -->
+    <figure data-dom-diff="typography-syntax-diff">
+        <figcaption>diff Syntax Highlighting Example</figcaption>
+        {% code 'diff' %}
+        -        Index: languages/ini.js
+        -        ===================================================================
+        -        --- languages/ini.js    (revision 199)
+        -        +++ languages/ini.js    (revision 200)
+        -        @@ -1,8 +1,7 @@
+        -         hljs.LANGUAGES.ini =
+        -         {
+        -           case_insensitive: true,
+        -        -  defaultMode:
+        -        -  {
+        -        +  defaultMode: {
+        -             contains: ['comment', 'title', 'setting'],
+        -             illegal: '[^\\s]'
+        -           },
+        -
+        -        *** /path/to/original timestamp
+        -        --- /path/to/new      timestamp
+        -        ***************
+        -        *** 1,3 ****
+        -        --- 1,9 ----
+        -        + This is an important
+        -        + notice! It should
+        -        + therefore be located at
+        -        + the beginning of this
+        -        + document!
+        -
+        -        ! compress the size of the
+        -        ! changes.
+        -
+        -          It is important to spell
+        -      {% endcode %}
+    </figure>
+</section>
+
+<section><!-- Syntax Highlighting -->
```

Comment:
Remove section. There is no Shadow DOM that needs testing.

Reply:
removed